packages/salesforcedx-vscode-apex-replay-debugger/src/index.ts

```diff
@@ -142,9 +145,26 @@ function registerDebugHandlers(): vscode.Disposable {
           );
           if (sfdxApex && sfdxApex.exports) {
             const lineBpInfo = await sfdxApex.exports.getLineBreakpointInfo();
+            let fsPath: string | undefined;
+            if (
+              vscode.workspace.workspaceFolders &&
+              vscode.workspace.workspaceFolders[0]
+            ) {
+              fsPath = vscode.workspace.workspaceFolders[0].uri.fsPath;
+            }
+            const config = vscode.workspace.getConfiguration();
+            const checkpointsEnabled = config.get(
+              'salesforcedx-vscode-apex-replay-debugger-checkpoints.enabled',
+              false
+            );
+            const returnArgs: LineBreakpointEventArgs = {
+              lineBreakpointInfo: lineBpInfo,
+              // for the moment always send undefined if checkpoints aren't enabled.
+              projectPath: checkpointsEnabled ? fsPath : undefined
```

Comment:
This is what I'd mentioned above. When checkpoints are a real boy and no longer need the check this will just return the project path, if there is one. On the adapter side nothing will need to change.